vb-be/src/modules/auth/auth.middleware.ts

```diff
@@ -7,17 +7,40 @@ import { ErrorHandler, ErrorText } from 'src/util/error';
 import { AuthService } from '@modules/auth/auth.service';
 import { TokenTypes } from '@modules/auth/auth.constants';
 
+/**
+ * Validates access token with type, JWT, and redis
+ * appends decoded token to headers, or throws 403
+ */
 @Injectable()
 export class ValidateAccessTokenMiddleware implements NestMiddleware {
+  constructor(private readonly authService: AuthService) {}
+
   async use(req: Request, res: Response, next: NextFunction) {
     const accessToken = req.headers['v-at'];
 
+    // validate access token is in header & is correct type
     if (!accessToken || typeof accessToken !== 'string') {
       return res
         .status(HttpStatus.Error)
         .json({ error: ErrorText.InvalidDataSet });
     }
-    next();
+
```

Comment:
I transferred this logic back over to the middleware since i realized it's going to be used by all of our private routes